src/sentry/utils/snuba.py

```diff
@@ -43,7 +43,7 @@ def timer(name, prefix='snuba.client'):
 )
 
 
-def raw_query(start, end, groupby, conditions=None, filter_keys=None,
+def raw_query(start, end, groupby=None, conditions=None, filter_keys=None,
```

Comment:
Trying to note that the aggregate columns aren't required here, otherwise it implies they're neccesary.